ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly-4.14-upgrade-from-stable-4.13.yaml

```diff
@@ -218,6 +376,41 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-ingress-custom-controller
+- as: azure-ipi-mini-perm-arm-f28
+  cron: 20 14 22 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-minimal-permission
+- as: azure-ipi-oidc-mini-perm-arm-f28
+  cron: 11 1 11 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      OCP_ARCH: arm64
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-cco-manual-workload-identity-minimal-permission
+- as: azure-ipi-ovn-etcd-encryption-compact-arm-f28
+  cron: 15 4 9 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: Standard_D4ps_v5
+      OCP_ARCH: arm64
+      SIZE_VARIANT: compact
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-ovn-etcd-encryption
 - as: azure-ipi-ovn-ipsec-arm-mixarch-day2-f28
```

Comment:
Could you please help remove this job?

Reply:
Need to remove both `azure-ipi-oidc-mini-perm-arm-f28` and `azure-ipi-ovn-etcd-encryption-compact-arm-f28`

Reply:
for this comment, I mean `azure-ipi-ovn-ipsec-arm-mixarch-day2-f28`, because we have other job covered `cucushift-installer-rehearse-azure-ipi-ovn-ipsec`